libfwupdplugin/fu-common.c

```diff
@@ -2000,6 +2000,25 @@ fu_byte_array_append_uint32 (GByteArray *array, guint32 data, FuEndianType endia
 	g_byte_array_append (array, buf, sizeof(buf));
 }
 
+/**
+ * fu_byte_array_set_size:
+ * @array: a #GByteArray
+ * @length:  the new size of the GByteArray
+ * @fillchar: the byte value to use when expanding the array
+ *
+ * Sets the size of the GByteArray, expanding it if necessary.
+ *
+ * Since: 1.5.0
+ **/
+void
+fu_byte_array_set_size (GByteArray *array, guint length, guint8 fillchar)
```

Comment:
Is there really a use case to fill the grown area with anything other than `NULL`?

Reply:
Flash is normally initialized with 0xff, but I agree in this case it's 100% redundant. Shall I remove `fillchar`?

Reply:
Yes

Reply:
Done.